Jellyfin.Api/Controllers/DynamicHlsController.cs

```diff
@@ -1797,13 +1797,17 @@ private string GetVideoArguments(StreamState state, int startNumber, bool isEven
 
         var args = "-codec:v:0 " + codec;
 
+        var requestedRange = state.GetRequestedRangeTypes(codec);
+
         if (string.Equals(state.ActualOutputVideoCodec, "h265", StringComparison.OrdinalIgnoreCase)
             || string.Equals(state.ActualOutputVideoCodec, "hevc", StringComparison.OrdinalIgnoreCase)
             || string.Equals(codec, "h265", StringComparison.OrdinalIgnoreCase)
             || string.Equals(codec, "hevc", StringComparison.OrdinalIgnoreCase))
         {
             if (EncodingHelper.IsCopyCodec(codec)
                 && (state.VideoStream.VideoRangeType == VideoRangeType.DOVI
+                    || (state.VideoStream.VideoRangeType == VideoRangeType.DOVIWithHDR10 && requestedRange.Contains(VideoRangeType.DOVIWithHDR10.ToString(), StringComparison.OrdinalIgnoreCase))
+                    || (state.VideoStream.VideoRangeType == VideoRangeType.DOVIWithHLG && requestedRange.Contains(VideoRangeType.DOVIWithHLG.ToString(), StringComparison.OrdinalIgnoreCase))
                     || string.Equals(state.VideoStream.CodecTag, "dovi", StringComparison.OrdinalIgnoreCase)
                     || string.Equals(state.VideoStream.CodecTag, "dvh1", StringComparison.OrdinalIgnoreCase)
                     || string.Equals(state.VideoStream.CodecTag, "dvhe", StringComparison.OrdinalIgnoreCase)))
```

Comment:
We can drop these now.
```suggestion
```

Reply:
ffmpeg currently does not support those profiles (handling metdata)... Profile 10 for AV1 and Profile 10 for 3D.

Reply:
Currently, the support for DV in FFmpeg is limited to P5/7/8 implemented through reverse engineering. Additional metadata will be ignored, so setting dvh1 here won't help.